packages/venia-ui/lib/components/AuthModal/authModal.js

```diff
@@ -1,51 +1,27 @@
-import React, { useCallback, useEffect, useState } from 'react';
+import React from 'react';
 import { func, shape, string } from 'prop-types';
-import { useUserContext } from '@magento/peregrine/lib/context/user';
 
 import { mergeClasses } from '../../classify';
 import CreateAccount from '../CreateAccount';
 import ForgotPassword from '../ForgotPassword';
 import MyAccount from '../MyAccount';
 import SignIn from '../SignIn';
 import defaultClasses from './authModal.css';
-
-const UNAUTHED_ONLY = ['CREATE_ACCOUNT', 'FORGOT_PASSWORD', 'SIGN_IN'];
+import { useAuthModal } from '@magento/peregrine/lib/mixins/AuthModal/useAuthModal';
```

Comment:
`mixins` has been renamed to `talons` 💪 🐦 💪 
```suggestion
import { useAuthModal } from '@magento/peregrine/lib/talons/AuthModal/useAuthModal';
```